src/sidebar/components/ShareDialog/FileInput.tsx

```diff
@@ -27,7 +27,6 @@ export default function FileInput({
         disabled={disabled}
         className="invisible absolute w-0 h-0"
         aria-hidden
-        tabIndex={-1}
```

Comment:
This was actually wrong. I added in a blind attempt to fix another test failing, but it turned out to be something completely unrelated, and forgot to undo this change.

`tabIndex={-1}` here means "this component is not part of the tab sequence, but can be programmatically focused", which is incorrect. It's a hidden component that will never be focused.

Reply:
When I looked at this it wasn't immediately obvious to me whether the `invisible` class (`visibility: hidden`) removes the element from the focus order. The tests on https://fuzzbomb.github.io/accessibility-demos/visually-hidden-focus-test.html confirm that it does though.